### PR TITLE
Added step in behat tests to wait for modal to be gone

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- see https://github.com/SU-SWS/stanford-caravan/pull/8
- Use the step definition

# Need Review By (Date)
- asap

# Urgency
- medium

# Steps to Test

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: tests/behat/features/Paragraphs/WYSIWYG.feature [hunk 1]
```diff
@@ -21,6 +21,7 @@ Feature: WYSIWYG Paragraph
     And I wait 1 seconds
     Then I fill in wysiwyg "Body" with "../../assets/documents/WYSIWYG.html"
     Then I press "Continue"
```
Comment (jbickar) [inline tests/behat/features/Paragraphs/WYSIWYG.feature]: Not within the scope of this PR, but why are these "Press" and not "Click"?

Comment (pookmish) [reply tests/behat/features/Paragraphs/WYSIWYG.feature]: button = "press" vs link = "click"

Comment (jbickar) [reply tests/behat/features/Paragraphs/WYSIWYG.feature]: Right. Then why not the `'Given I press the "<name>" button'` step definition from the Behat Drupal Extension? This isn't custom, is it?

Comment (pookmish) [reply tests/behat/features/Paragraphs/WYSIWYG.feature]: its not custom. i think its just shorthand for the same step

Comment (jbickar) [reply tests/behat/features/Paragraphs/WYSIWYG.feature]: Interdasting.

Comment (pookmish) [reply tests/behat/features/Paragraphs/WYSIWYG.feature]: i do wonder if `Then I press the "Continue" button` would've fixed the failures. :man_shrugging:

**Comment by pookmish** [comment]
Green lights

**Pull request merged**